Kitodo/src/main/java/org/kitodo/config/enums/ParameterCore.java

```diff
@@ -46,6 +46,12 @@ public enum ParameterCore implements ParameterInterface {
      */
     DIR_RULESETS(new Parameter<UndefinedParameter>("directory.rulesets")),
 
+    /**
+     * Absolute path to the directory that the ocr workflow files will be
+     * read from. It must be terminated by a directory separator ("/").
+     */
+    DIR_OCR_WORKFLOWS(new Parameter<UndefinedParameter>("directory.ocr.workflows")),
```

Comment:
You are defining here a new ` kitodo_config.properties` entry but did not provide an adjusted configuration file nor a documentation for this kind of entry.